### PR TITLE
add stack script

### PR DESCRIPTION
**Pull request by gorazdko**
## Abstract

Successfully tested on a linode:

> Onion service up and running: f5lg4vcjeskqgyigpqdxybcsg4kffjx52fgxlftefjmoeypcuz2zyvid.onion

### CODE REVIEW

**Comment by gorazdko** [comment]
Fixed.

@ChristopherA  can you re-review?

File: StackScript/torgap-demo.sh [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+# password you give it in the arguments.
+
+# torgap-demo.sh will create two logs in your root directory, to read them run:
+# $ cat standup.err
+# $ cat standup.log
+
+# This block defines the variables the user of the script needs to input
+# when deploying using this script.
+#
+# <UDF name="hostname" label="Short Hostname" example="Example: bitcoincore-testnet-pruned"/>
```
Comment (yancyribbens) [inline StackScript/torgap-demo.sh]: I'm not sure I understand why hostname and fqdn must be user created here.

File: src/main.rs [hunk 2]
```diff
@@ -3,15 +3,43 @@ extern crate log;
...
+use std::path::PathBuf;
 fn main() {
     let host = "127.0.0.1";
-    let port = "5555";
+    let port = "5556";
 
-    let server = Server::new(|request, mut response| {
+    let mut path = PathBuf::new();
+
+    match env::current_exe() {
```
Comment (yancyribbens) [inline src/main.rs:18]: I'm a little confused what's happening here.  It looks like a binary is being passed from the environment?

**Review by yancyribbens — APPROVED** [review]
Worked for me \o/

File: StackScript/torgap-demo.sh (hunk 1 above)
Comment (gorazdko) [reply StackScript/torgap-demo.sh]: tnx, these are remnants from the previous script. I removed them now.

File: src/main.rs (hunk 2 above)
Comment (gorazdko) [reply src/main.rs:18]: The path of executable is passed. So i can determine relative to it the paths of my html and text files.
This is just to ensure that running the server with `cargo run` or with ./executable behaves in the same way.

This code will be replaced by a more appropriate server soon as discussed, i.e. a static web server without the backend code.

**Comment by gorazdko** [comment]
@yancyribbens thank you for testing and review. I've addressed your comments.

**Pull request merged**